README.md

```diff
@@ -78,6 +83,7 @@ $ export TF_VAR_grafana_admin_password="SECRETpass"
 5. Deploy the dev configuration:
 
 ```
+bash-4.4# helm update
```

Comment:
should be `helm repo update`

Reply:
yes - per the latest helm `helm update` has been deprecated in favor of `helm repo update`

Reply:
fixed